src/actions/publish.js

```diff
@@ -5,31 +5,31 @@
 // @flow
 import { stripIndent } from 'common-tags';
 
-import { uploadBinaryProfileData } from '../profile-logic/profile-store';
-import { sendAnalytics } from '../utils/analytics';
+import { uploadBinaryProfileData } from 'firefox-profiler/profile-logic/profile-store';
+import { sendAnalytics } from 'firefox-profiler/utils/analytics';
 import {
   getUploadGeneration,
   getSanitizedProfile,
   getSanitizedProfileData,
   getRemoveProfileInformation,
   getPrePublishedState,
-} from '../selectors/publish';
+} from 'firefox-profiler/selectors/publish';
 import {
   getDataSource,
   getProfileNameForStorage,
   getUrlPredictor,
-} from '../selectors/url-state';
+} from 'firefox-profiler/selectors/url-state';
 import {
   getProfile,
   getZeroAt,
   getCommittedRange,
   getProfileFilterPageData,
-} from '../selectors/profile';
-import { viewProfile } from './receive-profile';
```

Comment:
For example this is one example of an import that is happening on the same file with `./`. We want them to stay the same. Could you revert this change and the others in the PR?